net/tstun/wrap.go

```diff
@@ -721,3 +732,15 @@ func (t *Wrapper) InjectOutbound(packet []byte) error {
 func (t *Wrapper) Unwrap() tun.Device {
 	return t.tdev
 }
+
+var (
+	metricPacketIn              = clientmetric.NewGauge("tstun_in_from_wg")
+	metricPacketInDrop          = clientmetric.NewGauge("tstun_in_from_wg_drop")
```

Comment:
Maybe s/in/recv/ and s/out/send/ throughout?

Reply:
Oh, I see this got discussed extensively above. I'm still tempted by send_to and recv_from, but am ok not attempting to repaint the shed.

Reply:
I'm fine with `tstun_send` and `tstun_recv` too (instead of out and in, respectively) but I imagine @danderson would find those ambiguous for the same reason as above.